libraries/ui-library/src/components/six-error/six-error.scss

```diff
@@ -0,0 +1,5 @@
+.six-error-container {
+  text-align: left;
+  color: var(--six-color-red);
```

Comment:
The color is now defined twice, once here as `--six-color-red`  and once in `form-control.tsx` as `--six-color-danger-800`. I think the color in `form-control.tsx` should be removed and be defined here as
`--six-color-danger-800`. `--six-color-red` is meant to be used as the SIX brand color.